source/_components/homekit_controller.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: "Homekit controller support"
+description: "Instructions how to integrate your Homekit devices within Home Assistant."
+date: 2018-03-19 21:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: apple-homekit.png
+ha_category: Hub
+ha_release: 0.66
+ha_iot_class: "Local Polling"
+---
+
+[HomeKit](https://developer.apple.com/homekit/) controller integration for Home Assistant allows you to connect HomeKit accessories to Home Assistant. This component should not be confused with the [HomeKit](homekit) component, which allows you to control Home Assistant devices via HomeKit.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Light](../light.homekit_controller)
+
+The component will be automatically configured if the [`discovery:`](components/discovery/) component is enabled. For each detected Homekit accessory, a configuration prompt will appear in the web front end. Use this to provide the HomeKit PIN. It can be disabled by adding the following configuration:
```

Comment:
Also, what about those that don't use `discovery`? Please specify the minimum config for this component

Reply:
The metadata needed for HomeKit is only present in the mdns broadcasts, so it requires discovery.